MOBT-1-Clean-Code/MOBT-1-Clean-Code/ViewController.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ViewController.swift
+//  MOBT-1-Clean-Code
+//
+//  Created by Mykyta Babanin on 11.03.2022.
+//
+
+import UIKit
+import Foundation
+
+class ViewController: UIViewController {
+    
+    // swiftlint:disable colon
```

Comment:
For single-disable please prefer `// swiftlint:disable:next rule_name` instead of the disable/enable pair